docs/gallery/domain/plot_behavior.py

```diff
@@ -0,0 +1,415 @@
+"""
+.. _behavior_basics:
+
+Behavior Data
+==================================
+
+This tutorial will demonstrate the usage of the :py:class:`~pynwb.behavior` module for adding
+behavior data to an :py:class:`~pynwb.file.NWBFile`.
+
+An :py:class:`~pynwb.file.NWBFile` represents a single session of an experiment.
+It contains all the data of that session and the metadata required to understand the data.
+
+
+.. seealso::
+    You can learn more about the :py:class:`~pynwb.file.NWBFile` format in the :ref:`basics` tutorial.
+
+
+The following examples will reference variables that may not be defined within the block they are used in. For
+clarity, we define them here:
+"""
+from datetime import datetime
+
+import numpy as np
+from dateutil import tz
+from pynwb.misc import IntervalSeries
+
+from pynwb.epoch import TimeIntervals
+
+from pynwb import NWBFile, TimeSeries, NWBHDF5IO
+from pynwb.behavior import (
+    SpatialSeries,
+    BehavioralTimeSeries,
+    Position,
+    BehavioralEvents,
+    CompassDirection,
+    BehavioralEpochs,
+    PupilTracking,
+    EyeTracking,
+)
+
+####################
+# Create an NWB File
+# ------------
+#
+# Create an :py:class:`~pynwb.file.NWBFile` object with the required fields
+# (``session_description``, ``identifier``, ``session_start_time``) and additional metadata.
+
+session_start_time = datetime(2018, 4, 25, 2, 30, 3, tzinfo=tz.gettz("US/Pacific"))
+
+nwbfile = NWBFile(
+    session_description="Mouse exploring an open field",  # required
+    identifier="Mouse5_Day3",  # required
+    session_start_time=session_start_time,  # required
+    session_id="session_1234",  # optional
+    experimenter="My Name",  # optional
+    lab="My Lab Name",  # optional
+    institution="University of My Institution",  # optional
+    related_publications="DOI:10.1016/j.neuron.2016.12.011",  # optional
+)
+
+nwbfile
+
+####################
+# Position
+# ------------
+#
+# Spatial Series
+# ^^^^^^^^^^^^^^^
+#
+# :py:class:`~pynwb.behavior.SpatialSeries` is a subclass of :py:class:`~pynwb.base.TimeSeries`
+# that represents the spatial direction, e.g., of gaze or travel, or position of an animal over time.
+#
+# .. seealso::
+#    You can learn more about the :py:class:`~pynwb.behavior.SpatialSeries` data type and
+#    :py:class:`~pynwb.behavior.Position` interface in the :ref:`basic_spatialseries` tutorial.
```

Comment:
Reading through the tutorial, I think it will be easier for the reader if the ``.. seealso::`` is at the end of the ``Spatial Series`` section., i.e., move it to before the "Create a Behavior Processing Module" heading.